content/practices/outcome-oriented-roadmap/index.md

```diff
@@ -0,0 +1,225 @@
+---
+# remove "draft:true" when ready for final publication 
+draft: false
+title: "Outcome-Oriented Roadmap"
+linkTitle: "Outcome Oriented Roadmap"
+description: "A communication and planning tool that shows sequential outcomes that help execute your strategy and build towards your vision."
+tags: ["Discovery", "Framing"]
+length: "2+ hours (block enough time or consider breaking the activity into two sessions)"
+participants: "Product Manager(s), core team, stakeholders"
+# custom "cover" image example: "boris/boris.png"
+image: "default-cover.png" 
+lastmod: "2021-02-25"
+why: 
+- Align our day-to-day work with the product vision and strategy
+- Keep focused on one goal at a time, and move metrics incrementally
+- Prioritize based on business & user value
+- Establish and maintain stakeholder and cross-team alignment, collaboration and visibility
+- Create a backlog
+
+when:
+- You’ve started development but don’t have a roadmap
+- During development, you need a way to align your backlog with your goals and metrics
+- Product sponsors and stakeholders ask for more visibility into the product direction and your progre
+
+what:
+- Whiteboard
+- Post-it notes
+- Sharpies
+- Painter’s tape
```

Comment:
These are great tools for in-person. Is there a space to suggest a remote set of tools. This works really well using Miro as a tool and the article mentions zoom rooms and such for remote implementations elsewhere. The article could set that up in tools, so the readers don't dismiss it early on.

Reply:
This is a really good point. Eric C. brought up that most of the practices / workshops read very "in-person" since they were originally developed with colocation in mind. 

The only thing we currently have is the ability to reference a public Miro template, such as the "remote" section of https://tanzu.vmware.com/developer/practices/goals-anti-goals/ 

We should have a better way to describing the remote-first variation of a practice or workshop. I've created issue #584 as a place to discuss this.

Reply:
Is it enough to add "For remote: Miro" for now or similar?

Reply:
Done!